modules/apps/asset/asset-tags-navigation-web/src/main/resources/content/Language_ca.properties

```diff
@@ -1,4 +1,4 @@
-asset-type-asset-count-help=El nombre de continguts amb aquesta etiqueta només tindrà en compte els continguts del tipus seleccionat.
+asset-type-asset-count-help=El nombre de continguts amb aquesta etiqueta només tindrà en compte els continguts del tipus seleccionat.
 javax.portlet.title.com_liferay_asset_tags_navigation_web_portlet_AssetTagsCloudPortlet=Núvol d'etiquetes
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.